src/deploy-actions.js

```diff
@@ -64,6 +66,8 @@ async function deployActions (config, deployConfig = {}, logFunc) {
     aioLogger.debug('Trimming out the manifest\'s actions...')
     filterEntities = undefined
     const builtActions = []
+    // this is a little weird, we are getting the list of built actions from the dist folder
```

Comment:
shall we log an issue for this tech debt?

Reply:
https://github.com/adobe/aio-lib-runtime/issues/194